swarm/storage/ldbstore.go

```diff
@@ -730,6 +731,28 @@ func (s *LDBStore) tryAccessIdx(ikey []byte, index *dpaDBIndex) bool {
 	return true
 }
 
+func (s *LDBStore) GetSchema() (string, error) {
```

Comment:
It is not schemas, but versions

`GetVersion` and `PutVersion` functions could just work with integers

Reply:
Schema is a `named database` (https://www.postgresql.org/docs/8.2/static/ddl-schemas.html) that we make no implicit assumptions about. If we have two schemas `a` and `b`, just by the names we cannot know if they are compatible or not, and we have to think how to migrate from one to the other, which is precisely what I want with this. If we have three versions 0.1, 0.2, and 0.3, we already assign semantics, and this limits our choices for migrations.